File: src/main/java/com/shabb/pongfinal/controls/WallControl.java
package com.shabb.pongfinal.controls;

import com.jme3.bullet.collision.shapes.CollisionShape;
import com.jme3.bullet.control.RigidBodyControl;

public class WallControl extends RigidBodyControl {
    public WallControl(CollisionShape shape) {
        super(shape);

        //-- Setup object as static
        setMass(0f);
        setKinematic(false);
        //-- Do not slow the ball when sliding against the floor.
        setFriction(0f);
        setRestitution(1f);
    }
}

File: src/main/java/com/shabb/pongfinal/controls/WallControl.kt
package com.shabb.pongfinal.controls

import com.jme3.bullet.collision.shapes.CollisionShape
import com.jme3.bullet.control.RigidBodyControl

class WallControl(shape: CollisionShape) : RigidBodyControl(shape) {
    init {
        //-- Setup object as static
        setMass(0f)
        isKinematic = false
        //-- Do not slow the ball when sliding against the floor.
        friction = 0f
        restitution = 1f
    }
}
